Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,10 @@
+{
+  "loc.friendlyName": "Download secure file",
```

Comment:
Is this resource file reflecting what's the goal of the package? I don't think so.

All of those should either be removed or updated or marked as issues to be fixed by someone who is speaking French, Spanish etc.

What we can do is to translate English and establish the base here.

Reply:
good point.

Reply:
Will be done in: https://github.com/aochmann/azure-download-multiple-secure-files/pull/2